tests/common/devices/sonic_asic.py

```diff
@@ -419,7 +421,10 @@ def command(self, cmdstr):
         if not self.sonichost.is_multi_asic or self.namespace == DEFAULT_NAMESPACE:
             return self.sonichost.command(cmdstr)
 
-        cmdstr = "sudo ip netns exec {} {}".format(self.namespace, cmdstr)
+        if new_format:
```

Comment:
Since this change will go to 202405 branch
Could you please confirm the new format works well on single ASIC devices?

Reply:
@yejianquan passed single-asic 202405 image